news/136.bugfix.4.rst

```diff
@@ -0,0 +1 @@
+Fix the two "largest allocating locations" sections in the ``memray stats`` report to actually aggregate by location. Previously they were aggregating by distinct stack, so if the same line of code was reached by two different paths, it would be counted separately instead of together.
```

Comment:
```suggestion
Fix the two "largest allocating locations" sections in the ``memray stats`` report to actually aggregate by location. Previously they were aggregating by distinct stacks, so if two different paths reached the same line of code, it would be counted separately instead of together.
```